hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/config/NetworkAdminServiceConfig.java

```diff
@@ -28,4 +28,5 @@
 @ConfigData("networkAdmin")
 public record NetworkAdminServiceConfig(
         @ConfigProperty(defaultValue = "/opt/hgcapp/services-hedera/HapiApp2.0/data/upgrade/current")
-                String upgradeArtifactsPath) {}
+                String upgradeArtifactsPath,
+        @ConfigProperty(defaultValue = "10") int maxTokensForAccountInfo) {}
```

Comment:
I think you should use existing config for now. I think we will move service specific configs to its services in the future. It is inside `TokensConfig` .

Reply:
done thank you